src/NumericalAlgorithms/Spectral/Spectral.hpp

```diff
@@ -92,20 +109,26 @@ constexpr size_t minimum_number_of_points(
  * \details Since Gauss-Lobatto quadrature has points on the domain boundaries
  * it must have at least two collocation points. Gauss quadrature can have only
  * one collocation point.
+ *
+ * \details For `CellCentered` the minimum number of points is 1.
  */
 template <Basis basis, Quadrature quadrature>
 constexpr size_t minimum_number_of_points =
     detail::minimum_number_of_points(basis, quadrature);
 /*!
  * \brief Maximum number of allowed collocation points.
  *
- * Since we cache points using compile time lookup we are limited to 10000
- * points for FiniteDifference. However, realistically 25 points is sufficient
- * for efficient parallelization and for DG-subcell.
+ * \details We choose a limit of 23 FD grid points because for DG-subcell the
+ * number of points in an element is `2 * (number_dg_points - 1)`. There is no
+ * way of generically retrieving the maximum number of grid points for non-FD,
+ * and thus we need to hard-code the value as well. If the number of grid points
```

Comment:
[nitpick] I think the paragraph would flow better with a slight rephrase of this sentence to "Because there is no way ..., we need to hard-code both values here."